crowbar_framework/lib/crowbar/upgrade.rb

```diff
@@ -52,6 +54,22 @@ def knife_files
           file_path = new_file
         end
 
+        if file.basename.to_s =~ /^bc-network-(.*)\.json$/
+          json = JSON.load(file.read)
+          # override with migrated attributes
+          json["attributes"] = SchemaMigration.migrated_attributes("network", json)
+          Tempfile.open("network") do |content|
+            content.write JSON.pretty_generate(json["attributes"])
```

Comment:
You probably just need to reassigne the result of migrated_attributes to a new Hash.

```
attributes_migrated = {}
attributes_migrated["attributes"] = SchemaMigration.migrated_attributes("network", json)
```
 and then ```JSON.pretty_generate(attributes_migrated)```

I haven't tried that though.